src/test/java/io/github/janmalch/kino/security/PasswordManagerTest.java

```diff
@@ -41,10 +41,19 @@ void isSamePassword_withCorrectSaltAndPassword() {
   }
 
   @Test
-  void isSamePassword_withIncorrectSaltAndPassword() {
+  void isSamePassword_withIncorrectPassword() {
     byte[] salt = pm.generateSalt();
     String hash = pm.hashPassword("testPassword", salt);
 
     assertFalse(pm.isSamePassword("testWrongPassword", hash, salt));
   }
+
+  @Test
+  void isSamePassword_withIncorrectSalt() {
+    byte[] salt = pm.generateSalt();
+    String hash = pm.hashPassword("testPassword", salt);
+    System.out.println(hash);
```

Comment:
In Zukunft so etwas rausnehmen vorher.
Können es jetzt drin lassen und irgendwann mal in einem branch so style Sachen aufräumen